src/hooks/api/tokens/useRefreshToken.ts

```diff
@@ -3,29 +3,30 @@ import { usePromisifiedMutation } from 'hooks/usePromisifiedMutation';
 import { graphql } from 'relay-runtime';
 import { useRefreshTokenMutation } from '__generated__/useRefreshTokenMutation.graphql';
 
-export function useRefreshToken() {
-  const [refreshTokenMutate] = usePromisifiedMutation<useRefreshTokenMutation>(graphql`
-    mutation useRefreshTokenMutation($id: DBID!) {
-      refreshToken(tokenId: $id) {
-        ... on RefreshTokenPayload {
-          __typename
-          token {
-            id
+export function useRefreshToken(): [(tokenId: string) => Promise<void>, boolean] {
```

Comment:
all that changed in this file was returning the `isRefreshing` state so callers don't have to manage the state themselves.